src/app/auth/components/sobv-text-box/sobv-text-box.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="sobv-text-box form-outline mb-2" >
+  <label class="form-label" [for]="title">{{ title }}</label>
+  <input class="form-control" [type]="type" [id]="title" [name]="title" [formControl]="control" />
```

Comment:
not correct behavior and naming convantion [for] and id should be same
 <label class="form-label" [for]="id">{{ label }}</label>
`<input class="form-control" [type]="type" [id]="id" [name]="control" [val]="val" [formControl]="control" />`